### ISSUE TITLE
Status cleared when sharing screen

### ISSUE THREAD

**Issue by Divi** [yakshaG/google-meet-slack-integration#6]
Hello again!

I have an issue when I start my screen share.
The status is cleared, but the listeners seem to be still active.

Here are the worker logs when I start the screen share:

```
Successfully intercepted network request. Clearing slack status
background.js:70 Removing meeting exit listener
background.js:144 Fetch finished loading: POST "https://slack.com/api/users.profile.set".
(anonymous) @ background.js:144
background.js:146 Slack status altered
background.js:19 Registering query tabs listener
background.js:22 Registering meeting exit listener
```

When I start a screen share, there is a request to `https://meet.google.com/$rpc/google.rtc.meetings.v1.MeetingDeviceService/CreateMeetingDevice`, I think if you add this to the join listener URLs filter, it should remove the Slack status, but will put it back just after.
It's not the best solution, but I can't find the network call that triggers the exit listener.

Thanks!

**Comment by aditya-67** [comment]
Hey @Divi 

Thank you for this! 

We just pushed the latest code to the repo that will solve this issue and #5. Please allow us some time to push it to the official Chrome Web store.

**How we did it?**

We solved #5 by saving the existing status of a user in Chrome storage and setting it back on exit. Please note that this requires `read:user:profile` scope of Slack to read the existing status. You can find the snippet [here](https://github.com/yakshaG/google-meet-slack-integration/blob/main/background.js#L168-L228).

Regarding the current issue, we saw a lot of inconsistent behavior when trying to intercept join and exit requests. We finally settled on intercepting the audio file(.ogg) requests that Google Meet makes on join and exit. This has proven to be a better way to take a stab at this problem.

<img width="814" alt="Screenshot 2022-08-08 at 7 16 56 PM" src="https://user-images.githubusercontent.com/9695866/183433421-836e0128-b912-4b3c-9be1-db76c08da710.png">

**Known issue**

> The join and exit events are fired based on network requests for the sound file, which fetches the .ogg file. So if the interval between two join events or two exit events is less than 30 seconds (approx), the browser uses a cached sound file. So super short meetings will be affected. Seemed like a good trade-off, since we get unique endpoints for each event.


While we prepare to push to the official extension, you can clone the current repo and test it on your local by loading the package.

Looking forward to more feedback and thanks for taking the time to contribute! We really appreciate it 🙌🏻

**Comment by yakshaG** [comment]
The update is live on chrome store with version number 0.0.15. Do let us know if there are any issues.
